relay-server/src/metrics_extraction/snapshots/relay_server__metrics_extraction__event__tests__extract_span_metrics_all_modules.snap

```diff
@@ -1262,6 +1264,7 @@ expression: metrics
             "resource.render_blocking_status": "blocking",
             "span.category": "resource",
             "span.description": "https://cdn.domain.com/path/to/file-hk2YHeW7Eo2XLCi*z*KuljsgCAD6hyWCyOYZM.css",
+            "span.domain": "cdn.domain.com",
```

Comment:
Ideally this should be `*.domain.com` to be consistent with HTTP spans. I will clean this up in a follow-up PR, I think resource and HTTP spans can share the same code path.